content/outcomes/application-development-rhythm/foundations.md

```diff
@@ -0,0 +1,208 @@
+---
+title: Foundations
+weight: 10
+layout: single
+team:
+  - VMware Tanzu Labs
+---
+This article is the first in a five-part series.
+
+Before you explore this series,
+it is necessary you understand the context of
+*modern application principles* and *practices*.
+
+This article outlines the context,
+as well as the core building blocks necessary to sustain a product or
+project using modern application development practices.
+
+## What you will learn
+
+In this article you will be able to:
+
+-   Explain the context of modern application development.
+-   The types of building blocks that comprise modern application
```

Comment:
Is this "Explain the types" or "Describe the types" or "List the types"?

Reply:
@joemoore Describe the types.  Thanks for the catch.